tests/test_root_commands.py

```diff
@@ -132,12 +141,81 @@ def complete(partialPath):
     assert complete('t') == ['tags.', 'theme.', 'tmp.']
     assert complete('') == [l + '.' for l in hlwm.list_children_via_attr('')]
 
-@pytest.mark.parametrize('attrtype', ['int', 'bool', 'string', 'color', 'uint'])
+
+@pytest.mark.parametrize('attrtype', ATTRIBUTE_TYPES)
 @pytest.mark.parametrize('name', ['my_test', 'my_foo'])
 @pytest.mark.parametrize('object_path', ['', 'clients', 'theme.tiling.active'])
-def test_user_attributes_without_removal(hlwm, attrtype, name, object_path):
+def test_new_attr_without_removal(hlwm, attrtype, name, object_path):
     path = (object_path + '.' + name).lstrip('.')
 
     hlwm.call(['new_attr', attrtype, path])
 
     hlwm.get_attr(path)
+
+
+@pytest.mark.parametrize('attrtype', ATTRIBUTE_TYPES)
+def test_new_attr_existing_attribute(hlwm, attrtype):
```

Comment:
Is there any better name for "not a user attribute"? Maybe "static", or "builtin"?

In order to avoid confusion with "existing user attribute", I'd rather avoid using "existing" to refer to "builtins" as well.